samples/src/main/scala/SparkStreamingKustoSink.scala

```diff
@@ -19,40 +21,56 @@ object SparkStreamingKustoSink {
       .getOrCreate()
 
     // read messages from Azure Event Hub
-    val connectionString = ConnectionStringBuilder("Event Hub Connection String")
-      .setEventHubName("Event Hub Name")
+    val connectionString = ConnectionStringBuilder("Endpoint=sb://bittonhub.servicebus.windows.net/;SharedAccessKeyName=KustoReadKey;SharedAccessKey=PSPCQgAtQGS02sQ77PxT+lJDBRvcnrlx4IZJP+vSep0=")
+      .setEventHubName("ehbitton")
       .build
 
     val eventHubsConf = EventHubsConf(connectionString)
-      .setStartingPosition(EventPosition.fromEndOfStream)
+      .setStartingPosition(EventPosition.fromEndOfStream).setMaxEventsPerTrigger(5)
+
+    val r = scala.util.Random
 
-    val eventhubs = spark.readStream
-      .format("eventhubs")
-      .options(eventHubsConf.toMap)
-      .option("checkpointLocation", "/checkpoint")
-      .load()
+    var s = r.nextInt
+    if(s < 0){
+      s = s * (-1)
+    }
+    var customSchema = StructType(Array(
+      StructField("IDGC", StringType, true)))
+
+    val eventhubs = spark.readStream.
+      schema(customSchema).
+      csv("C:\\Users\\ohbitton\\IdeaProjects\\azure-kusto-spark\\samples\\data")
 
     val toString = udf((payload: Array[Byte]) => new String(payload))
-    val df = eventhubs.withColumn("body", toString(eventhubs("body")))
+    import spark.implicits._
+    //    val df = eventhubs.select($"body" cast "string")
+
+    spark.conf.set("spark.sql.streaming.checkpointLocation", s"target/temp/checkpoint/$s")
 
-    spark.conf.set("spark.sql.streaming.checkpointLocation", "target/temp/checkpoint/")
-    spark.conf.set("spark.sql.codegen.wholeStage", "false")
+
+    val authority = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+    //    spark.conf.set("spark.sql.codegen.wholeStage", "false")
 
     // Write to a Kusto table from a streaming source
-    val df1 = df
+    val df1 = eventhubs
       .writeStream
       .format("com.microsoft.kusto.spark.datasink.KustoSinkProvider")
-      .option(KustoOptions.KUSTO_CLUSTER, "Your Kusto Cluster")
-      .option(KustoOptions.KUSTO_DATABASE, "Your Kusto Database")
-      .option(KustoOptions.KUSTO_TABLE, "Your Kusto Destination Table")
-      .option(KustoOptions.KUSTO_AAD_CLIENT_ID, "Your Client ID")
-      .option(KustoOptions.KUSTO_AAD_CLIENT_PASSWORD, "Your secret")
-      .option(KustoOptions.KUSTO_WRITE_ENABLE_ASYNC, "true")
+      .option(KustoOptions.KUSTO_CLUSTER, "ohbitton")
```

Comment:
Right, also not here... actually it is problematic to even have it on GH in review since this is visible to all